_posts/2019-08-08-hacking-for-cruise-control.md

```diff
@@ -234,7 +244,9 @@ spec:
   selector:
     name: my-cruise-control
 ```
+
 Apply to the cluster:
+
 ```bash
 kubectl apply -f cruise-control-deployment.yaml
 ```
```

Comment:
There is a typo near the bottom of the post that I missed too "Vist 'http://127.0.0.1:9090' in browser",  could you fix that while you are in here?